kobo/apps/organizations/admin.py

```diff
@@ -20,6 +20,7 @@ class OwnerInline(BaseOwnerInline):
 
 class OrgUserInline(admin.StackedInline):
     model = OrganizationUser
+    raw_id_fields = ("user",)
```

Comment:
Nice! One last thing: this little 'View on site' link goes to a dead page:

![image](https://github.com/kobotoolbox/kpi/assets/7819986/00ccaeeb-bede-4df5-b080-f000bf128d29)


```suggestion
    raw_id_fields = ("user",)
    view_on_site = False
```

Once that's done, please go ahead and merge👍